src/components/Button/index.js

```diff
@@ -254,47 +258,41 @@ class Button extends Component {
                 onPressOut={this.props.onPressOut}
                 onMouseDown={this.props.onMouseDown}
                 disabled={this.props.isLoading || this.props.isDisabled}
-                style={[
+                wrapperStyle={[
                     this.props.isDisabled ? {...styles.cursorDisabled, ...styles.noSelect} : {},
+                    (this.props.isDisabled && (this.props.success || this.props.danger)) ? styles.buttonOpacityDisabled : undefined,
+                    (this.props.isDisabled && !this.props.danger && !this.props.success) ? styles.buttonDisabled : undefined,
```

Comment:
Why we are moving those from inner style to wrapper style?

Reply:
that's how the new component works, it'll make sense when you try it out.

Reply:
Can you please elaborate? Those styles were being passed to `<OpacityView />` why we are passing them to `<Pressable />`